### PR TITLE
update hot fix cudf link v21.12.2

### PR DESCRIPTION
**Pull request by nvliyuan**
Signed-off-by: liyuan <yuali@nvidia.com>

update hot fix cudf link v21.12.2

### CODE REVIEW

File: docs/download.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ for your hardware's minimum driver version.
 ### Download v21.12.0
 * Download the [RAPIDS
   Accelerator for Apache Spark 21.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/21.12.0/rapids-4-spark_2.12-21.12.0.jar)
-* Download the [RAPIDS cuDF 21.12.0 jar](https://repo1.maven.org/maven2/ai/rapids/cudf/21.12.0/cudf-21.12.0-cuda11.jar)
+* Download the [RAPIDS cuDF 21.12.2 jar](https://repo1.maven.org/maven2/ai/rapids/cudf/21.12.2/cudf-21.12.2-cuda11.jar)
```
Comment (tgravescs) [inline docs/download.md:46]: configs.md, getting started and others also reference the 21.12.0 cudf jar, are we going to update those as well?

Comment (nvliyuan) [reply docs/download.md:46]: updated

**Comment by tgravescs** [comment]
build

**Review by jlowe — DISMISSED** [review]

**Review by tgravescs — DISMISSED** [review]

**Review by abellina — APPROVED** [review]

**Review by abellina — CHANGES_REQUESTED** [review]
Does pom.xml need updating to cudf 21.12.2?

**Comment by nvliyuan** [comment]
> Does pom.xml need updating to cudf 21.12.2?

updated cudf.version in pom file

**Review by tgravescs — APPROVED** [review]

**Review by abellina — APPROVED** [review]

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Pull request merged**